demos/TransferLearning/TransferLearningDemo.py

```diff
@@ -101,89 +37,53 @@ class Demo:
     # Common accuracy metric for all outputs, but can use different metrics for different output
     LOSS_METRICS = ['accuracy']
 
-    IMAGE_SIZE = 160  # All images will be resized to 160x160
+    SGD = optimizers.SGD(lr=0.01, decay=1e-6, momentum=0.9, nesterov=True)
+
+    IMAGE_SIZE = 224
     IMAGE_SHAPE = (IMAGE_SIZE, IMAGE_SIZE, 3)
 
-    batch_size = 32
+    ngraph_bridge = None
+
+    batch_size = 16
     epochs = 2
     fine_tune_at = 100
```

Comment:
I think these vars should be dropped because the `__init__` is setting these values.